src/interfaces/keyIterator.ts

```diff
@@ -7,6 +7,8 @@ export interface KeyIterator {
   subType?: 'seed' | 'private-key'
   /** The wallet native SDK instance, if any exists */
   walletSDK?: any
+  /** The wallet controller, for the hardware wallets only */
+  controller?: any
```

Comment:
Why any?

Reply:
You're connect, there is a type for this, fixed in https://github.com/AmbireTech/ambire-common/pull/944/commits/d343850ac2ccafd5a05aa3e6ddb6e5bcdc6b1c70